modules/webp-uploads/load.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * WebP-Uploads Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ *
+ * Module Name: WebP uploads.
+ * Description: Uses WebP as the default format for new JPEG image uploads if the server supports it.
+ * Focus: images
+ * Experimental: No
+ */
+
+// Only enable if the server supports WebP.
+add_filter( 'image_editor_output_format', 'webp_default_filter_image_editor_output_format', 10, 3 );
+
+/**
+ * Filter the image editor default output format mapping.
+ *
+ * For uploaded JPEG images, map the default output format to WebP.
+ *
+ * @since 1.0.0
+ *
+ * @param string $output_format The image editor default output format mapping.
+ * @param string $filename      Path to the image.
+ * @param string $mime_type     The source image mime type.
+ * @return string The new output format mapping.
+ */
+function webp_default_filter_image_editor_output_format( $output_format, $filename, $mime_type ) {
```

Comment:
Let's rename the prefix to `webp_uploads`, in accordance with how the module slug was changed.

Reply:
👍🏼